### PR TITLE
Improve UX

### PR DESCRIPTION
**Pull request by JoelForamitti**
- Move extensions and dashboard into settings section & tags into users section
- Button to create new users
- Better layout for displaying data in ObjectDetail

image

### CODE REVIEW

File: src/components/core/CoreSettings.vue [hunk 1]
```diff
@@ -23,9 +30,43 @@ for (const extension of extensions) {
...
+      </TabPanel>
+      <TabPanel :header="t('Extensions')">
+        <ObjectList :store="mainStore" :name="'extensionsExtensions'">
+          <template #item="slotProps">
+            <PrimePanel :header="slotProps.data.label" class="mb-5">
+              <p class="pb-3">{{ slotProps.data.description }}</p>
+              <p>Active: {{ slotProps.data.active }}</p>
+              <p>Built-in: {{ slotProps.data.built_in }}</p>
+              <p v-if="slotProps.data.version">
+                Version: {{ slotProps.data.version }}
```
Comment (nanu-c) [inline src/components/core/CoreSettings.vue:50]: That should be translated

File: src/components/datatable/ObjectDetail.vue [hunk 2]
```diff
@@ -57,40 +57,51 @@ const editActive = ref(false);
...
+          </div>
+          <div class="flex-auto break-words overflow-hidden">
+            <div v-if="data.detailLoaded" class="w-full">
+              <div v-if="field.input_type == 'multiselect'">
+                <div
+                  v-for="(value, i) in data.detail[name]"
+                  :key="i"
+                  class="bg-gray-200 text-gray-700 px-1.5 pt-1.5 pb-0.5 mb-1.5 whitespace-pre-wrap"
+                >
+                  {{ field.choices![value] }}
```
Comment (nanu-c) [inline src/components/datatable/ObjectDetail.vue:73]: can we translate it?

File: src/components/datatable/ObjectDetail.vue [hunk 3]
```diff
@@ -57,40 +57,51 @@ const editActive = ref(false);
...
-            <span v-else-if="data.detail[name]" class="data">
-              {{ data.detail[name] }}
-            </span>
-            <!-- No content -->
-            <span v-else-if="field.write_only" class="data">********</span>
+                {{ data.detail[name] }}
+              </div>
+
+              <div v-else-if="data.detail[name]" class="data break-words">
+                {{ data.detail[name] }}
```
Comment (nanu-c) [inline src/components/datatable/ObjectDetail.vue:93]: same with this?

File: src/components/datatable/ObjectDetail.vue [hunk 4]
```diff
@@ -112,7 +123,7 @@ const editActive = ref(false);
 </template>
 
 <style lang="scss" scoped>
-.data {
-  @apply bg-slate-200 mr-1 pt-1 px-1 h-6 rounded whitespace-pre-wrap leading-7;
-}
+// .data {
+//   @apply bg-slate-200 mr-1 pt-1 px-1 h-6 rounded whitespace-pre-wrap leading-7;
+// }
```
Comment (nanu-c) [inline src/components/datatable/ObjectDetail.vue:128]: This is commented code ;), pelase remove

**Review by nanu-c — CHANGES_REQUESTED** [review]

File: src/components/datatable/ObjectDetail.vue (hunk 4 above)
Comment (JoelForamitti) [reply src/components/datatable/ObjectDetail.vue:128]: done!

File: src/components/core/CoreSettings.vue (hunk 1 above)
Comment (JoelForamitti) [reply src/components/core/CoreSettings.vue:50]: done!

File: src/components/datatable/ObjectDetail.vue (hunk 2 above)
Comment (JoelForamitti) [reply src/components/datatable/ObjectDetail.vue:73]: hmm i think we first need to write a safe wrapper for the translate function, as the application breaks if it receives anything else than a string. have added this to the board

File: src/components/datatable/ObjectDetail.vue (hunk 3 above)
Comment (JoelForamitti) [reply src/components/datatable/ObjectDetail.vue:93]: same as above

**Review by nanu-c — APPROVED** [review]

**Pull request merged**